### PR TITLE
Mk private sector fixes 3

### PR DESCRIPTION
**Pull request by mkangia**
hopefully last set of fixes

best review 🐟 / 🐟

### CODE REVIEW

File: custom/enikshay/integrations/nikshay/field_mappings.py [hunk 1]
```diff
@@ -150,9 +155,10 @@
 }
 
 health_establishment_type = {
-    'hospital': 'H',
-    'lab': 'L',
+    'Hospital': 'H',
+    'Lab': 'L',
     'pharmacy': 'P',
```
Comment (calellowitz) [inline custom/enikshay/integrations/nikshay/field_mappings.py]: is this supposed to not be capitalized. Its the only one

**Comment by mkangia** [comment]
@calellowitz i need this in deploy and its a safe + independent change so i am merging this. Let me know if there is anything i should fix in this.

File: custom/enikshay/integrations/nikshay/field_mappings.py [hunk 2]
```diff
@@ -150,9 +155,10 @@
...
+    'Doctor': 'P'
```
Comment (calellowitz) [inline custom/enikshay/integrations/nikshay/field_mappings.py]: should this have a different letter (same as pharmacy now)

File: custom/enikshay/integrations/nikshay/field_mappings.py (hunk 1 above)
Comment (mkangia) [reply custom/enikshay/integrations/nikshay/field_mappings.py]: there is actually no use for that value.. we can remove it later.

File: custom/enikshay/integrations/nikshay/field_mappings.py (hunk 2 above)
Comment (mkangia) [reply custom/enikshay/integrations/nikshay/field_mappings.py]: Ya it should be the same.

**Pull request merged**